.github/workflows/conformance.yml

```diff
@@ -62,38 +68,34 @@ jobs:
           ref: "conformance/v0.5.0-skipped-tests"
           path: "gateway-api"
 
-      - name: Setup Goenv
-        uses: ./consul-api-gateway/.github/actions/goenv
+      - name: Setup Kind cluster
+        if: matrix.cluster-type == 'kind'
+        uses: ./consul-api-gateway/.github/actions/setup-kind
         with:
-          go-version: ${{ env.GO_VERSION }}
-
-      - name: Build binary
-        env:
-          CGO_ENABLED: "0"
-          GOARCH: "amd64"
-          GOOS: "linux"
-        working-directory: "consul-api-gateway"
-        run: go build -o ./consul-api-gateway
+          metallb-config-path: "./consul-api-gateway/internal/testing/conformance/metallb-config.yaml"
 
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1
+      - name: Generate Unique Cluster Name
+        if: matrix.cluster-type == 'eks'
+        id: unique
+        run:
+            echo "::set-output name=clustername::$( echo consul-api-gateway-test-${{ github.run_number }}-$RANDOM )"
```

Comment:
I was curious what other things might be available on the context to concoct a unique cluster name.

`${{ github.run_id }}-${{ github.run_number }}-${{ github.run_attempt }}` was the thing I came up with. The only real benefit is if we ever needed to determine the cluster name based off of a given Actions run

Reply:
Also wondering if we can either use this same cluster name for AWS, GKE and Azure or, if not, can we encapsulate it into the `setup-eks` action

Reply:
The issue I was running into with the eks specific stuff, is we have multiple jobs kicking off due to the matrix that seem to share run-number/run-attempt getting name conflicts because it was requesting all at the same time.